### PR TITLE
Update test case results for tfsec with new container

### PR DESCRIPTION
**Pull request by yi2020**

### CODE REVIEW

File: test-cases/terraform/aws/encryption/at-rest/athena_not_encrypted/tfsec_results.txt [hunk 1]
```diff
@@ -1,21 +1,22 @@
 
 Problem 1
 
-  [AWS002][ERROR] Resource 'aws_s3_bucket.hoge' does not have logging enabled.
-  /src/main.tf:1-3
+  [AWS059][ERROR] Resource 'aws_athena_database.hoge' .
```
Comment (yi2020) [inline test-cases/terraform/aws/encryption/at-rest/athena_not_encrypted/tfsec_results.txt:4]: @owenrumney this seems like a bug in tfsec? (no description of the issue)

File: test-cases/terraform/aws/logging/eks_logging_disabled/tfsec_results.txt [hunk 2]
```diff
@@ -1,85 +1,9 @@
...
+  running checks       1.186ms
   files loaded         1
-  blocks               5
-  evaluated blocks     5
-  modules              0
-  module blocks        0
-  ignored checks       0
 
-4 potential problems detected.
+No problems detected!
```
Comment (yi2020) [inline test-cases/terraform/aws/logging/eks_logging_disabled/tfsec_results.txt:8]: @owenrumney I think something may be broken - this is a test case that the previous docker image caught, and this one doesn't.

**Comment by owenrumney** [comment]
Yup, looks like it's my bad, I missed a docker container name https://github.com/iacsecurity/tool-compare/blob/2b84a946176b5de7f7f7dbaa2bfd532139a262dd/run_all_tools.sh#L37 -

in the `docker run` it still has `liamg/tfsec`. Want me to raise a new PR or can you add it to this one?

**Comment by yi2020** [comment]
@owenrumney I got it.

**Comment by yi2020** [comment]
@owenrumney using the tfsec/tfsec container there's a bunch of test cases that now are caught by tfsec that weren't earlier, good job :) 

Anything else before I merge this PR?

**Comment by owenrumney** [comment]
@yi2020 nope, all good

**Comment by yi2020** [comment]
Closes #8

**Pull request merged**